tools/odbc/duckdb_odbc.cpp

```diff
@@ -164,8 +166,8 @@ void OdbcHandleStmt::FillIRD() {
 			new_record.SetSqlDescType(sql_type);
 		}
 
-		new_record.sql_desc_type_name = col_type.ToString();
-		duckdb::ApiInfo::GetColumnSize<SQLINTEGER>(col_type, &new_record.sql_desc_display_size);
+		new_record.sql_desc_type_name = duckdb::TypeIdToString(col_type.InternalType());
```

Comment:
Is this the physical type? If it's the logical type, you can use `EnumUtil::ToString(col_type));`.